src/govuk/components/button/button.test.js

```diff
@@ -59,8 +59,8 @@ describe('/components/button', () => {
      *
      * Examples don't do this and we need it to have something to submit
      */
-    async function trackClicks () {
-      page.evaluate(() => {
+    function trackClicks (page) {
```

Comment:
With (potentially) multiple pages or tabs open at once, we need to pass in which one we use